src/main/java/com/powsybl/balances_adjustment/pevf/PevfExchangesXml.java

```diff
@@ -0,0 +1,323 @@
+            switch (reader.getLocalName()) {
+                case RESOLUTION_ELEMENT_NAME:
+                    String resolution = reader.getElementText();
+                    spacing.set(Duration.parse(resolution));
+                    break;
+
+                case TIME_INTERVAL_ELEMENT_NAME:
+                    readTimeInterval(reader, TIME_INTERVAL_ELEMENT_NAME, start, end);
+                    break;
+
+                case POINT_ELEMENT_NAME:
+                    readPoint(reader, positions, quantities);
+                    break;
+
+                default:
+                    LOGGER.warn(UNEXPECTED_TOKEN, reader.getLocalName());
+            }
+        });
+    }
+
+    private static void readTimeInterval(XMLStreamReader reader, String rootElement, AtomicReference<ZonedDateTime> start, AtomicReference<ZonedDateTime> end) throws XMLStreamException {
+        XmlUtil.readUntilEndElement(rootElement, reader, () -> {
+            switch (reader.getLocalName()) {
+                case START_ELEMENT_NAME :
+                    String periodStart = reader.getElementText();
+                    start.set(ZonedDateTime.parse(periodStart));
+                    break;
+
+                case END_ELEMENT_NAME :
+                    String periodEnd = reader.getElementText();
+                    end.set(ZonedDateTime.parse(periodEnd));
+                    break;
+
+                default:
+                    LOGGER.warn(UNEXPECTED_TOKEN, reader.getLocalName());
+            }
+        });
+    }
+
+    private static void readPoint(XMLStreamReader reader, LinkedList<Integer> positions, LinkedList<Double> quantities) throws XMLStreamException {
+        XmlUtil.readUntilEndElement(POINT_ELEMENT_NAME, reader, () -> {
+            switch (reader.getLocalName()) {
+                case POSITION_ELEMENT_NAME:
+                    String position = reader.getElementText();
+                    positions.add(Integer.parseInt(position));
+                    break;
+
+                case QUANTITY_ELEMENT_NAME:
+                    String quantity = reader.getElementText();
+                    quantities.add(Double.parseDouble(quantity));
+                    break;
+
+                default:
+                    LOGGER.warn(UNEXPECTED_TOKEN, reader.getLocalName());
+            }
+        });
+    }
+
+    private static void readReason(XMLStreamReader reader, AtomicReference<String> code, AtomicReference<String> text) throws XMLStreamException {
+        XmlUtil.readUntilEndElement(REASON_ELEMENT_NAME, reader, () -> {
+            switch (reader.getLocalName()) {
+                case CODE_ELEMENT_NAME:
+                    code.set(reader.getElementText());
+                    break;
+
+                case TEXT_ELEMENT_NAME:
+                    text.set(reader.getElementText());
+                    break;
+
+                default:
+                    LOGGER.warn(UNEXPECTED_TOKEN, reader.getLocalName());
```

Comment:
Same remark: maybe we should throw an exception (and explicitly skip certain values we don't want to parse)